sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -31,7 +31,7 @@ export function useFormatters() {
    * @param date string or date to be converted.
    * @returns string representation (e.g. Aug 05 2021).
    */
-  const dateOnlyLongString = (date: string | Date): string => {
+  const dateOnlyLongString = (date?: string | Date): string => {
```

Comment:
Why is date optional?

Reply:
The method was already prepared to deal with an undefined date but the parameter was not. I just change the parameter to match the method body.
Sometimes we receive the values, for instance, from API, and we do not need to check if a value is present or not.